idelib/dataset.py

```diff
@@ -1746,12 +1747,15 @@ def __getitem__(self, idx, display=False):
 
         if isinstance(idx, (int, np.integer)):
 
+
             if idx >= len(self):
                 raise IndexError("EventArray index out of range")
 
             if idx < 0:
                 idx = max(0, len(self) + idx)
 
+            return self.arraySlice(idx, idx + 1)[:, 0]
```

Comment:
Is this in the wrong place/indentation level, or should all the code after it be removed?

Reply:
Code after should be removed, I just got side-tracked.